pkg/ui/workspaces/cluster-ui/webpack.config.js

```diff
@@ -84,7 +86,10 @@ module.exports = {
             options: {extension: ".module.scss"},
           },
         ],
-        exclude: /node_modules/,
+        exclude: [
+          /node_modules/,
+          /db-console\/src\/js/,
```

Comment:
I'm curious, what is this "exclude" doing? Isn't `db-console` living in a different workspace?

Reply:
"exclude" instructs to ignore from preprocessing by specific rule (in our case by Babel) under specified paths. Modules under these paths WILL be included in final bundle anyway if they're referenced somewhere in the code.

You're right it lives in different workspace, but `cluster-ui` refers to `protobufs` bundle (which lives under `db-console` workspace) as a linked dependency in `package.json`:
```
"@cockroachlabs/crdb-protobuf-client": "link:../db-console/src/js",
```
Webpack resolves this a relative path so it has to be excluded as relative path as well.